public/spec/email/spec.md

```diff
@@ -183,16 +183,16 @@ Both POP3 and IMAP services can be defined as follows.
 ```ballerina
 email:Service emailObserver = service object {
 
-   remote function onMessage(email:Message emailMessage) {
+   remote method onMessage(email:Message emailMessage) {
```

Comment:
All changes in this file seem to be changes to code snippets. Should remain as `function` in code snippets.

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/6013.